lsp-mode.el

```diff
@@ -795,6 +796,7 @@ Changes take effect only when a new session is started."
     ("^PKGBUILD$" . "shellscript")
     ("^go\\.mod\\'" . "go.mod")
     ("^settings.json$" . "jsonc")
+    ("^yang\\.settings$" . "jsonc")
```

Comment:
@jcs090218 should this be `"^yang\\.settings\\'"`? What's the meaning of `\\'` usage?

Also shouldn't `"^settings.json$"` be `"^settings\\.json$"`?

Reply:
> should this be `"^yang\\.settings\\'"`? What's the meaning of `\\'` usage?

`\\'` matches the end of a string; `$` matches the end of line. So both work.

> Also shouldn't `"^settings.json$"` be `"^settings\\.json$"`?

Yes, you are right.